OutOfSchool/OutOfSchool.WebApi/Services/StatisticService.cs

```diff
@@ -113,11 +113,11 @@ public async Task<IEnumerable<DirectionStatistic>> GetPopularDirections(int limi
         }
 
         /// <inheritdoc/>
-        public async Task<IEnumerable<WorkshopDTO>> GetPopularWorkshops(int limit)
+        public async Task<IEnumerable<WorkshopCard>> GetPopularWorkshops(int limit)
         {
             logger.Information("Getting popular workshops started.");
 
-            var workshops = workshopRepository.Get<int>();
+            var workshops = workshopRepository.Get<int>(includeProperties: "Address,Direction");
```

Comment:
can we use something like a $"{nameof(Address)},{nameof(Direction)}" instead of string literal ?